src/main/java/org/jabref/logic/externalfiles/LinkedFileHandler.java

```diff
@@ -38,7 +38,7 @@ public LinkedFileHandler(LinkedFile fileEntry,
         this.databaseContext = Objects.requireNonNull(databaseContext);
         this.filePreferences = Objects.requireNonNull(filePreferences);
     }
-
+    
```

Comment:
If possible, remove the spaces - they are not needed here. Maybe, <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>L</kbd> helps to reformat (and delete thus spaces) easily.

Reply:
I agree with you. That's my mistake.